src/Modules/Extensions/templates/Backend/Actions/ThemeIndex.html.twig

```diff
@@ -20,45 +20,45 @@
       </div>
       <div class="card-body">
         <div id="installedThemes" class="row row-cols-1 row-cols-sm-2 row-cols-md-3 row-cols-lg-2 row-cols-xl-3 g-4">
-          {% for installedThemeData in installedThemes %}
-            {% set installableTheme = installedThemeData.theme %}
-            {% set activateForm = installedThemeData.activateForm %}
+          {% for installed_theme_data in installedThemes %}
```

Comment:
can you update the php so that it sets installed_themes?

Reply:
@bjorvack this has not been processed?

Reply:
fixed 23a8a55c5f99cf0b12513ebefef3175d2e4176cd